### PR TITLE
Add training details

### PR DESCRIPTION
**Pull request by kentang-mit**
In this PR, we add training details for the following model:
- camera-only detection
- camera-only BEV map segmentation
- LiDAR-only detection
- LiDAR-only BEV map segmentation

The results are also slightly improved compared with our previous release.

### CODE REVIEW

File: mmdet3d/datasets/nuscenes_dataset.py [hunk 1]
```diff
@@ -267,10 +267,14 @@ def get_data_info(self, index: int) -> Dict[str, Any]:
                 camera2ego[:3, 3] = camera_info["sensor2ego_translation"]
                 data["camera2ego"].append(camera2ego)
 
-        # TODO (Haotian): test set submission.
+                # camera to lidar transform
+                camera2lidar = np.eye(4).astype(np.float32)
+                camera2lidar[:3, :3] = camera_info["sensor2lidar_rotation"]
+                camera2lidar[:3, 3] = camera_info["sensor2lidar_translation"]
+                data["camera2lidar"].append(camera2lidar)
```
Comment (zhijian-liu) [inline mmdet3d/datasets/nuscenes_dataset.py:274]: Could you double-check whether the other transformations are still actively being used? If not, we could remove them.

File: mmdet3d/models/fusion_models/bevfusion.py [hunk 2]
```diff
@@ -5,16 +5,18 @@
 from torch import nn
 from torch.nn import functional as F
 
+# from mmdet3d.core import bbox3d2result, merge_aug_bboxes_3d
```
Comment (zhijian-liu) [inline mmdet3d/models/fusion_models/bevfusion.py:8]: Remove the commented code.

**Review by zhijian-liu — APPROVED** [review]

File: mmdet3d/models/fusion_models/bevfusion.py (hunk 2 above)
Comment (kentang-mit) [reply mmdet3d/models/fusion_models/bevfusion.py]: Done.

File: mmdet3d/datasets/nuscenes_dataset.py (hunk 1 above)
Comment (kentang-mit) [reply mmdet3d/datasets/nuscenes_dataset.py:274]: Sure, we can remove this in a future version with coordinate system reformatting.

**Pull request merged**